pkg/controller/batchrelease/batchrelease_plan_executor.go

```diff
@@ -110,12 +110,13 @@ func (r *Executor) executeBatchReleasePlan(workloadController workloads.Workload
 		needed, err := workloadController.IfNeedToProgress()
 		switch {
 		case err != nil:
-			setCondition(status, "VerifyWorkloadError", err.Error(), v1.ConditionFalse)
+			setCondition(status, "Verifying", v1.ConditionFalse, "Failed", err.Error())
```

Comment:
can we enumerate all valid condition using variables and give each meaningful comments ?

Reply:
sure... sorry... I forgot to do that ...